src/localization/finnish.ts

```diff
@@ -5,16 +5,16 @@ export var finnishSurveyStrings = {
   pageNextText: "Seuraava",
   completeText: "Valmis",
   previewText: "Esikatselu",
-  editText: "Muokata",
-  startSurveyText: "alkaa",
-  otherItemText: "Muu (kuvaile)",
+  editText: "Muokkaa",
+  startSurveyText: "Aloita",
+  otherItemText: "Muu (tarkenna)",
```

Comment:
_Tarkenna_ ("specify") seems more appropriate than _kuvaile_ ("describe, illustrate, characterize") here. The latter verb is something one would use in contexts like "Describe yourself as a team member" where one is looking for vague impressions and broad strokes rather than a specific answer.